.github/workflows/yamcs-quickstart-e2e.yml

```diff
@@ -7,30 +7,48 @@ on:
       - opened
 
 jobs:
-  e2e-couchdb:
-    if: ${{ github.event.label.name == 'pr:e2e:quickstart' }}
+  yamcs-quickstart-e2e:
+    timeout-minutes: 5
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: "16"
       - uses: actions/checkout@v3
         with:
           repository: yamcs/quickstart
           ref: master
-      - run: make all
+          path: 'quickstart'
+      - name: trigger make all from docker dir. Orphan and redirect output to docker/makeout.txt
+        run: |
+          cd quickstart/docker
+          touch makeout.txt
+          make all &> makeout.txt &
+          cd ../..
+      - run: ls -latR
+      - name: Archive makeout.txt
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          path: quickstart/docker/makeout.txt
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
       - run: npm install
-      - run: npx playwright@1.25.2 install
-      - run: ls -latr
+      - run: npx playwright@1.25.2 install chromium
       - run: npm run build:example
       - run: npm run test:getopensource
       - run: npm run test:e2e:quickstart
+      - name: Archive makeout.txt #not working due to pathing
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          path: /quickstart/docker/makeout.txt
```

Comment:
Remove the prepended `/` here

Reply:
even that doesn't help. There's something being destroyed when cloning the second time